apps/console/src/configs/routes.tsx

```diff
@@ -1270,158 +1270,90 @@ export const getAppViewRoutes = (): RouteInterface[] => {
         }
     ];
 
-    if (legacyMode?.apiResourcesV1) {
-        defaultRoutes.unshift({
-            category: "console:develop.features.sidePanel.categories.application",
-            children: [
-                {
-                    component: lazy(() =>
-                        import("@wso2is/admin.api-resources.v1/pages/api-resource-edit")
-                    ),
-                    exact: true,
-                    id: "apiResources-edit",
-                    name: "extensions:develop.sidePanel.apiResources",
-                    path: APIResourcesConstants.getPaths().get("API_RESOURCE_EDIT"),
-                    protected: true,
-                    showOnSidePanel: false
-                }
-            ],
-            component: lazy(() =>
-                import("@wso2is/admin.api-resources.v1/pages/api-resources")
-            ),
-            exact: true,
-            icon: {
-                icon: getSidePanelIcons().apiResources
+    defaultRoutes.unshift({
```

Comment:
We can directly include these two in the array since we don't need to conditionally append now right?

Reply:
Fixed with [d0f8fab](https://github.com/wso2/identity-apps/pull/6487/commits/d0f8fab615c67a855bbf44a0fb3bc7a77aae9c94)